packages/editor/src/components/collab-sidebar/comment-form.js

```diff
@@ -47,10 +49,13 @@ function CommentForm( {
 	const inputId = useInstanceId( CommentForm, 'comment-input' );
 
 	return (
-		<>
-			<VisuallyHidden as="label" htmlFor={ inputId }>
-				{ __( 'Comment' ) }
-			</VisuallyHidden>
+		<VStack spacing="3">
+			<label
+				htmlFor={ inputId }
+				className="editor-collab-sidebar-panel__comment-label"
+			>
+				{ labelText || __( 'Comment' ) }
+			</label>
```

Comment:
Instead of using a label element, I think it would be better to reuse the VisuallyHidden component.

Reply:
+1.